model/aspnetcore/metrics.yaml

```diff
@@ -114,3 +125,166 @@ groups:
     attributes:
       - ref: aspnetcore.rate_limiting.result
         requirement_level: required
+
+  # blazor components
+  - id: metric.aspnetcore.components.navigation
+    type: metric
+    metric_name: aspnetcore.components.navigation
+    stability: development
+    brief: Total number of route changes.
+    instrument: counter
+    unit: "{route}"
+    note: |
+      Meter name: `Microsoft.AspNetCore.Components`; Added in: ASP.NET Core 10.0
+    extends: aspnetcore.components.metrics.attributes
+    attributes:
+      - ref: aspnetcore.components.type
+        requirement_level: required
+      - ref: aspnetcore.components.route
+        requirement_level: required
+
+  - id: metric.aspnetcore.components.event.duration
+    type: metric
+    metric_name: aspnetcore.components.event.duration
+    stability: development
+    brief: Duration of processing browser event.
+    instrument: histogram
+    unit: "s"
+    note: |
+      Meter name: `Microsoft.AspNetCore.Components`; Added in: ASP.NET Core 10.0
+    extends: aspnetcore.components.metrics.attributes
+    attributes:
+      - ref: aspnetcore.components.type
+        requirement_level: required
+      - ref: aspnetcore.components.method
+        requirement_level: required
+      - ref: aspnetcore.components.attribute.name
+        requirement_level: required
+
+  - id: metric.aspnetcore.components.event.exceptions
```

Comment:
this would be covered by the `aspnetcore.component.event.duration` once it includes `error.type`, so we probably don't need it

Reply:
Why is that better than separate metric ?